api/routes/batch/delete.go

```diff
@@ -0,0 +1,31 @@
+package batch
+
+import (
+	"context"
+	"github.com/arkhn/fhir-river/api/topics"
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/go-redis/redis"
+	"time"
+)
+
+// Delete deletes a batch by removing its Redis keys and Kafka topics
+func Delete(batchID string, rdb *redis.Client, admin *kafka.AdminClient) error {
```

Comment:
as I understand it, this is not exposed as a route right ? I'm a little bit confused between batch.Cancel and batch.Delete, the SOUND similar but `Cancel` is an HTTP handler acting as a wrapper around `Delete`. Maybe this function should not be exposed like this (I know it is used in `monitor` as well so it's not that simple). I'm sorry I don't have a clear solution right now.

Reply:
I agree it's confusing, I will think about it a bit more but no idea how to solve that so far